core/runtime.rs

```diff
@@ -801,7 +785,7 @@ impl JsRuntime {
     let mut state = state_rc.borrow_mut();
     let module_map = module_map_rc.borrow();
 
-    let has_pending_ops = !state.pending_ops.is_empty();
+    let has_pending_ops = state.pending_ops.len() > state.unrefed_ops.len();
```

Comment:
Suggestion: rename this to e.g. `has_pending_refed_ops` or something like that? Makes it a little clearer that the logic here is "if more pending ops than unref'd ops exist, we have ref'd ops."

Reply:
Ok